### PR TITLE
Run tests against docker containers

### PR DESCRIPTION
**Pull request by dacrome**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  services:
+    - docker
+  environment:
+    DOCKER_HOST: http://localhost:2375
+
+database:
+  pre:
+    - export DEBIAN_FRONTEND=noninteractive
+    - sudo service mysql stop && sudo service postgresql stop
+    - sudo apt-get -y remove mysql-server postgresql-9.4
```
Comment (tkrille) [inline circle.yml]: i think stopping is enough. don't waste precious test time :)

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    - sudo apt-get -y autoremove
```
Comment (tkrille) [inline circle.yml]: same as above

**Comment by tkrille** [comment]
I'm getting test failures when using the mysql profile:

```
Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.628 sec <<< FAILURE! - in org.osiam.client.CompleteUserIT
search_for_user_by_complex_query(org.osiam.client.CompleteUserIT)  Time elapsed: 0.436 sec  <<< FAILURE!
java.lang.AssertionError: 
Expected: is <1L>
     but: was <0L>
```

I think this is still the datetime bug.

File: circle.yml (hunk 1 above)
Comment (dacrome) [inline circle.yml]: haha sure, when they stopped, they not listening on the port anymore :)

**Comment by dacrome** [comment]
updated.

**Pull request merged**